requirements.txt

```diff
@@ -1,14 +1,14 @@
 absl-py
 packaging
 pandas
-tensorflow
+tf-nightly-cpu==2.16.0.dev20231103  # Pin a working nightly until rc0.
 tensorflow-datasets
 flake8
 regex
 isort
 black
 pytest
 pycocotools
-keras-core
 build
+keras-core>=0.1.6
```

Comment:
Wouldn't `tf-nightly` already pull in `keras 3` via `keras-nightly`.  Do we need to install `keras-core`?

Reply:
This is needed if users are using keras 2 and set a backend. we will remove this once keras 3 is released.